.buildkite/pipeline.yml

```
@@ -107,4 +107,44 @@ steps:
     depends_on:
       - lint
       - unit
-      - test_network
\ No newline at end of file
+      - test_network
+  - label: test_byoc_aws
+    key: test_byoc_aws
+    if: build.pull_request.labels includes 'byoc-ready'
+    command: make test_byoc_aws
+    plugins:
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - secret-id: sdlc/prod/buildkite/devex_ci_creds_aws
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
+      - docker#v5.11.0:
```

```suggestion
      - docker#v5.12.0:
```
use latest version [v5.12.0](https://github.com/buildkite-plugins/docker-buildkite-plugin/releases/tag/v5.12.0)